### PR TITLE
[XV UI] Add chrono

### PR DESCRIPTION
**Pull request by VelocityPulse**

### CODE REVIEW

File: XV/Assets/Scripts/UI/UIChrono.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	private void Start()
+	{
+		mStopWatch = new Stopwatch();
+
+		mMyText = GetComponent<Text>();
+		mMyText.text = "Chrono: " + "0.0s";
+		mStopWatch.Reset();
+
+		//Callback necessary in case stop doesnt make more than a frame
+		TimelineEvent.StopEvent += OnTimelineEventStop;
```
Comment (4quet) [inline XV/Assets/Scripts/UI/UIChrono.cs:26]: Event must be unregistered.
use `OnEnable()` and `OnDisable()`

File: XV/Assets/Scripts/UI/UIChrono.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	}
+
+	// Update is called once per frame
+	void Update()
+	{
+		UnityEngine.Debug.Log(TimelineManager.sGlobalState);
```
Comment (4quet) [inline XV/Assets/Scripts/UI/UIChrono.cs]: Remove `Debug.Log()`

**Review by 4quet — CHANGES_REQUESTED** [review]
See comments

**Review by banthony42 — APPROVED** [review]
checked.

**Review by 4quet — APPROVED** [review]
Check

**Pull request merged**